samples/ChangefeedMigrationSample/Program.cs

```diff
@@ -0,0 +1,193 @@
+﻿namespace ChangefeedMigrationSample
```

Comment:
Please add a header describing how to run this problem.  If it takes and command line options detail them here.  

You also should add the Apache 2.0 license header you see in other Microsoft sample code.